core/src/commonTest/kotlin/com/sourcepoint/mobile_core/network/SourcepointClientTest.kt

```diff
@@ -369,69 +365,13 @@ class SourcepointClientTest {
         assertTrue(response.gdpr?.acceptedCategories?.isEmpty() == true)
     }
 
-    @Test
-    fun postGDPRChoiceActionContainCorrectUrl() = runTest {
-        val mockEngine = mock("""{"uuid":"1"}""")
-        SourcepointClient(123, 321, "test", httpEngine = mockEngine).postChoiceGDPRAction(
-            SPActionType.AcceptAll,
-            GDPRChoiceRequest(
-                sendPVData = true,
-                propertyId = 123,
-                includeData = IncludeData(),
-                uuid = null,
-                messageId = null,
-                authId = null,
-                consentAllRef = null,
-                vendorListId = null,
-                pubData = null,
-                pmSaveAndExitVariables = null,
-                sampleRate = null,
-                idfaStatus = null,
-                granularStatus = null
-            )
-        )
-        val defaultRequest = DefaultRequest()
-        assertEquals(Url("https://cdn.privacy-mgmt.com/wrapper/v2/choice/gdpr/11?env="+defaultRequest.env+"&scriptType="
-                +defaultRequest.scriptType+"&scriptVersion="+defaultRequest.scriptVersion),
-            mockEngine.requestHistory.last().url
-        )
-    }
-
-    @Test
-    fun postGDPRChoiceActionContainCorrectBody() = runTest {
```

Comment:
@andresilveirah the same as above here. This unit test tests the absolute minimum for body to backend give as 200 response